ui/src/main.tsx

```diff
@@ -30,7 +31,7 @@ ReactDOM.createRoot(document.getElementById('root')!).render(
         <ColorSchemeScript defaultColorScheme="auto" />
         <MantineProvider defaultColorScheme="auto">
           <ModalsProvider>
-            <Notifications position="top-right" />
+            <Notifications position="top-center" zIndex={99999999} />
```

Comment:
Why 99999999?

Reply:
prevent vision block by the fixed style in the future, like:
<img width="705" alt="image" src="https://github.com/pingcap/LinguFlow/assets/11549583/288b7b13-98e8-4be7-a0c9-c090a16500a6">

Reply:
Notification should be at the top of all elements.

Reply:
But `99999999` is a magic number, should be replaced with project level const var to avoid someone use more `9` than you.